### PR TITLE
Feature/Chat Alpha

### PR DESCRIPTION
**Pull request by jakobuid**

### CODE REVIEW

File: chat/sample/src/main/kotlin/com/walletconnect/chatsample/ui/shared/ChatSharedViewModel.kt [hunk 1]
```diff
@@ -29,169 +29,135 @@ class ChatSharedViewModel : ViewModel() {
...
     }
 
     companion object {
-        private const val SWIFT_ENS = "Swift.eth"
-        private const val SWIFT_ACCOUNT_ID = "eip155:1:0xab16a96d359ec26a11e2c2b3d8f8b8942d5bfcdb"
-        private const val KOTLIN_ENS = "Kotlin.eth"
         private const val KOTLIN_ACCOUNT_ID = "eip155:2:0xab16a96d359ec26a11e2c2b3d8f8b8942d5bfcdb"
-        private const val JS_ENS = "Js.eth"
+        private const val SWIFT_ACCOUNT_ID = "eip155:1:0xab16a96d359ec26a11e2c2b3d8f8b8942d5bfcdb"
         private const val JS_ACCOUNT_ID = "eip155:3:0xab16a96d359ec26a11e2c2b3d8f8b8942d5bfcdb"
```
Comment (Elyniss) [inline chat/sample/src/main/kotlin/com/walletconnect/chatsample/ui/shared/ChatSharedViewModel.kt:160]: I think there will be a couple of bugs if we leave it. Maybe we should tell devs to add their addresses?

Comment (jakobuid) [reply chat/sample/src/main/kotlin/com/walletconnect/chatsample/ui/shared/ChatSharedViewModel.kt:160]: Yes I agree, gonna remove it and change a little bit the register logic. Not to register the same account with different pubKey but to check whether there is already a pubKey in the storage, if yes I'm gonna use it, if not I'm gonna create a new one

Comment (Elyniss) [reply chat/sample/src/main/kotlin/com/walletconnect/chatsample/ui/shared/ChatSharedViewModel.kt:160]: I like your proposal

Comment (Elyniss) [reply chat/sample/src/main/kotlin/com/walletconnect/chatsample/ui/shared/ChatSharedViewModel.kt]: I think we still have this issue of registering `eip155:2:0xab16a96d359ec26a11e2c2b3d8f8b8942d5bfcdb` too many times

**Review by Elyniss — APPROVED** [review]

**Review by Elyniss — APPROVED** [review]

File: chat/sample/src/main/kotlin/com/walletconnect/chatsample/ui/invites/InvitesFragment.kt [hunk 2]
```diff
@@ -16,16 +17,17 @@ import kotlinx.coroutines.flow.onEach
 class InvitesFragment : Fragment(R.layout.fragment_invites) {
     private val binding by viewBinding(FragmentInvitesBinding::bind)
     private val viewModel: ChatSharedViewModel by activityViewModels()
-    private val invitesAdapter by lazy { InvitesAdapter(viewModel::acceptRequest) }
+    private val invitesAdapter by lazy {
+        InvitesAdapter(onAccept = { chatUI -> viewModel.acceptInvitation(chatUI) }, onReject = { id -> viewModel.rejectInvitation(id) })
```
Comment (TalhaAli00) [inline chat/sample/src/main/kotlin/com/walletconnect/chatsample/ui/invites/InvitesFragment.kt:21]:
```suggestion
        InvitesAdapter(onAccept = viewModel::acceptInvitation, onReject = viewModel::rejectInvitation)
```

**Review by TalhaAli00 — APPROVED** [review]

**Pull request merged**